### DISCUSSION TITLE
NetCDF 4.8.1 Test Failure: FAIL

### DISCUSSION THREAD

**Discussion by SangamYa** [Unidata/netcdf-c#2807] [Q&A]
Hello,

I've encountered an issue while running 'make check' during the installation of netCDF v4.8.1. The test results show that one test, test_rcmerge, has failed with the following details:

netCDF 4.8.1: ncdump/test-suite.log
=========================================

# TOTAL: 26
# PASS:  25
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: test_rcmerge
==================

0a1,2
> |HTTP.NETRC|->|home/user3/.netrc|
> |HTTP.COOKIEJAR|->|home/user3/.urs_cookies|
FAIL test_rcmerge.sh (exit status: 1)
I would appreciate any suggestions or guidance on how to address this issue during the installation process. Thank you.

**Comment by WardF** [comment]
Can you provide the config.log file generated, as well as the test-suite.log file generated, when you built and ran the tests?

**Reply by SangamYa** [reply]
Yes sure, here I have attached the required files.
[test-suite.log](https://github.com/Unidata/netcdf-c/files/13533855/test-suite.log)
[config.log](https://github.com/Unidata/netcdf-c/files/13533856/config.log)
Thank you in advance

**Comment by WardF** [comment]
In trying to replicate this issue, I'm seeing errors related to the ncdap tests; is this the only test failure you're seeing?  The ncdap issues are not related to any bug, but rather to updates to our remote data servers since 4.8.1 was released. It is possible that this `rcmerge` test failure is related.
